script/worker_test

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+# Build the worker
+
+# Stop on errors
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ ! -d node_modules ]; then
```

Comment:
Locally this is not enough as package-lock might have changed. It's better to just not do this, and require people to call `script/bootstrap` in their flows when they need it.